src/components/sidebar.jsx

```diff
@@ -58,8 +58,8 @@ export default ({user, signOut, recentGroups}) => (
       </div>
       <div className='box-body'>
         <ul>
-          <li>News</li>
-          <li>Support</li>
+          <li><Link to='freefeed'>News</Link></li>
+          <li><Link to='support'>Support</Link></li>
```

Comment:
I know this will probably work, but it's better to have it rock-solid with a slash: `/freefeed`, `/support`. Let me know if I'm missing something.